packages/itwin/grouping-mapping-widget/src/widget/utils.ts

```diff
@@ -2,14 +2,60 @@
 * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
 * See LICENSE.md in the project root for license terms and full copyright notice.
 *--------------------------------------------------------------------------------------------*/
+import type { ReactElement } from "react";
+
 export type CreateTypeFromInterface<Interface> = {
   [Property in keyof Interface]: Interface[Property];
 };
 
-export function debounce<F extends (...args: any[]) => void>(f: F, delay: number) {
+export function debounce<F extends (...args: any[]) => void>(
+  f: F,
+  delay: number,
+) {
   let timer: number | undefined;
   return (...args: any[]) => {
     timer && window.clearTimeout(timer);
     timer = window.setTimeout(f, delay, ...args);
   };
 }
+
+/**
+ * Group Extension type definition
+ */
+export interface GroupExtension {
+  /**
+   * unique identifier of the extension
+   */
+  name: string;
+  /**
+   * display label in the widget
+   */
+  displayLabel: string;
+  /**
+   * UI Component interface
```

Comment:
This documentation is not enough. To me it is unclear what kind of UI this is and why must it be capitalized? Does all text in that component need to be in capital letters?